ci-operator/config/medik8s/common/new-release.sh

```diff
@@ -6,16 +6,23 @@ if [ "${PWD##*/}" != "common" ]; then
   exit 1
 fi
 
-# check number of args
-if [ $# -ne 2 ]; then
-  echo "Usage: $0 <repo> <branch>"
+# check the number of arguments
+if [ $# -lt 2 ] || [ $# -gt 3 ]; then
+  echo "Usage: $0 <repo> <branch> <operator_released_version>"
```

Comment:
maybe surround <operator_released_version> with [] to state that it's optional?

Reply:
SGTM